editor/src/clj/editor/resource.clj

```diff
@@ -108,29 +108,36 @@
   (atom {}))
 
 ;; The same logic implemented in Project.java.
-;; If you change something here, plese change it there as well
-;; Search for excluedFilesAndFoldersEntries.
+;; If you change something here, please change it there as well
+;; Search for excludedFilesAndFoldersEntries.
```

Comment:
Actually, the name of the field is misspelled in `Bob`. 😞

Reply:
oh no